httpclient/src/main/kotlin/tbdex/sdk/httpclient/TbdexHttpClient.kt

```diff
@@ -102,7 +90,18 @@ object TbdexHttpClient {
     val body: RequestBody = Json.stringify(CreateExchangeRequest(rfq))
```

Comment:
The line above this (it's out of my GitHub diff view here so having to comment here)

```kt
val path = "/exchanges/$exchangeId"
```

Should just be `/exchanges` right?

Reply:
fixed